source/advocacy.js

```diff
@@ -0,0 +1,66 @@
+let javascriptLoader = {
+  initialize: function () {
+    this.defer(this.loadOptInMonster, 'jQuery')
+
+    this.defer(this.loadCrossDomain, 'ga')
+  },
+
+  defer: function (a, d) {
```

Comment:
Good that you added this back in -- noticed it was missing on first perusal. Completely not sure if this is the best way to approach adding code on a delay, but works to ensure jQuery and ga functions are available.